test/services/webuntis/webuntisMetadataService.test.js

```diff
@@ -0,0 +1,289 @@
+const { expect } = require('chai');
+
+const app = require('../../../src/app');
+const testObjects = require('../helpers/testObjects')(app);
+
+const webuntisMetadataService = app.service('webuntisMetadata');
+const { webuntisMetadataModel } = require('../../../src/services/webuntis/model');
+const { datasourceRunModel } = require('../../../src/services/datasources/model');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['datasourceRunModel' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4899819)